src/streamlink/plugins/aliez.py

```diff
@@ -8,13 +8,15 @@
 from streamlink.stream import HTTPStream, RTMPStream
 
 _url_re = re.compile(r"""
-    http(s)?://(\w+\.)?aliez.tv
+    http(s)?://(\w+\.)?aliez.\w+/
```

Comment:
Agreed it's a bit too expanded. I didn't want to add a repeating group to the TLD (and consequently not accept .co.uk ect) in case other sites happen to use "aliez" as a subdomain. For example if twitch decided to use the following format:
http://aliez.twitch.tv/videos/
This plugin would match. I don't know of any that use this but I could see sites using profile names as subdomains and wouldn't want conflicts.